benchmarks/util.py

```diff
@@ -141,14 +141,10 @@ def get_cpu_name():
 
 
 def get_gpu_name():
-  gpu_names = subprocess.check_output(
+  return subprocess.check_output(
```

Comment:
Is this change intended or not? It seems to be reverting to older codes.

Reply:
You are right, thanks, I have reverted this line back.